### PR TITLE
[TFLite] Implemented ONE_HOT Operator for TFLite.

### PR DESCRIPTION
**Pull request by jainris**
* Added implementation for ONE_HOT Operator.
* Added tests for ONE_HOT Operator.

### CODE REVIEW

**Comment by leandron** [comment]
cc @anijain2305 @u99127 @mbaret @FrozenGene @tqchen

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -2886,6 +2887,56 @@ def convert_detection_postprocess(self, op):
...
+            from tflite.OneHotOptions import OneHotOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 4, "Input tensor's length should be 4"
+
+        # Ensuring input isn't quantized
+        for t in input_tensors:
+            assert not t.qnn_params, "Quantized input is not expected."
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py:2903]: we could use `assert all(not i.qnn_params for i in input_tensors)`

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -2886,6 +2887,56 @@ def convert_detection_postprocess(self, op):
...
+
+        # TFlite ONE_HOT requires both on_value
+        # and off_value, making dtype redundant.
+        indices = input_tensors[0]
+        depth = input_tensors[1]
+        on_value = input_tensors[2]
+        off_value = input_tensors[3]
+
+        assert on_value.tensor.Type() == off_value.tensor.Type(), \
+            "on_value and off_value should be of the same type"
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py:2913]: no `of`

**Comment by jainris** [comment]
"continuous-integration/jenkins/pr-merge" check failed. I don't think it is because of my changes. 
Trying a rerun of the checks.

**Comment by FrozenGene** [comment]
Thanks @jainris @leandron

**Pull request merged**